packages-proprietary/tokens/src/components/ams/date-input.tokens.json

```diff
@@ -2,7 +2,7 @@
   "ams": {
     "date-input": {
       "background-color": { "value": "{ams.color.background}" },
-      "box-shadow": { "value": "inset 0 0 0 {ams.border.width.s} {ams.color.interactive.secondary}" },
+      "border": { "value": "{ams.border.width.s} solid {ams.color.interactive.secondary}" },
```

Comment:
We could remove the color here, which makes the border the currentColor.

Reply:
I’m not sure this always works. Let’s see tomorrow.

Reply:
Updated to use `currentColor` as discussed.